app/Domain/Integrations/Repositories/IntegrationRepository.php

```diff
@@ -33,5 +34,6 @@ public function activateWithOrganization(UuidInterface $id, UuidInterface $organ
     public function approve(UuidInterface $id): void;
 
     /** @return Collection<Integration> */
-    public function getDraftsByTypeAndBetweenMonthsOld(IntegrationType $type, int $startMonths, int $endMonths): Collection;
+    public function getDraftsByTypeAndBetweenMonthsOld(IntegrationType $type, int $startMonths, int $endMonths, string $mailType): Collection;
```

Comment:
Here is a new variable name `mailType` but I think that is just the template name?

Reply:
I removed "mailType" and made it pass everywhere the enum TemplateName, I also call the variable this.